apps/web/pages/video/[uid].tsx

```diff
@@ -49,11 +58,30 @@ export default function JoinCall(props: JoinCallPageProps) {
       ...(typeof meetingPassword === "string" && { token: meetingPassword }),
     });
     callFrame.join();
+    callFrame.on("recording-started", onRecordingStarted).on("recording-stopped", onRecordingStopped);
     return () => {
       callFrame.destroy();
     };
   }, []);
 
+  const onRecordingStopped = () => {
+    const data = { recordingId: recordingId.current, bookingUID: booking.uid };
+
+    fetch("/api/sendDailyVideoRecordingEmail", {
+      method: "POST",
+      body: JSON.stringify(data),
+    }).catch((err) => {
+      console.log(err);
+    });
+
+    recordingId.current = null;
+  };
+
+  const onRecordingStarted = (event?: DailyEventObjectRecordingStarted | undefined) => {
+    const res = recordingStartedEventResponse.safeParse(event);
+    if (res.success) recordingId.current = res.data.recordingId;
```

Comment:
Do we expect a situation when in recording-started event `recordingId` won't be present? That seems weird to me. If we don't expect that we should simpy parse, so that error is thrown when such a situation arises.